app/javascript/components/mentoring/Discussion.tsx

```diff
@@ -10,6 +10,7 @@ import { AddDiscussionPost } from './discussion/AddDiscussionPost'
 type Links = {
   scratchpad: string
   exercise: string
+  posts: string
```

Comment:
Great! I hope the value of links passing becomes even more apparent now 🤗 (e.g. this change was minimal because all that's changing is where the link lives)

Reply:
@SleeplessByte Yes! It's really the way to go :)